project/Whitesource.scala

```diff
@@ -17,7 +17,7 @@ object Whitesource extends AutoPlugin {
         if (isSnapshot.value)
           if (gitCurrentBranch.value == "master") "master"
           else "adhoc"
-        else "stable"
+        else "2.5-stable"
```

Comment:
would be better to extract this from the `version`. Do we do that in some other place?

Reply:
We originally did that for whitesource ;), but then moved to this version scheme to be able to easily report on the 'latest supported' versions. Let me see if I can find an example.

Reply:
`def majorMinor(version: String): Option[String] ="""\d+\.\d+""".r.findFirstIn(version)`

Reply:
so something like `majorMinor(version.value).map(_ + "-stable").getOrElse("snapshot"))` (not sure whether 'snapshot' makes sense here anymore though)

Reply:
thanks, I was thinking we could keep `master` and `adhoc` without version info